src/sentry/models/exporteddata.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+
+import six
+from enum import Enum
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, JSONField, Model, sane_repr
+
+
+class ExportStatus(six.binary_type, Enum):
+    Early = "EARLY"  # The download is being prepared
+    Valid = "VALID"  # The download is ready for the user
+    Expired = "EXPIRED"  # The download has been deleted
+
+
+# TODO(Leander): Connect PostiveSmallIntegerField to enum
+# class ExportQueryType(Enum):
+
+
+class ExportedData(Model):
+    """
+    Stores references to asynchronous data export jobs being stored
+    in the Google Cloud Platform temporary storage solution.
+    """
+
+    __core__ = False
+
+    organization = FlexibleForeignKey("sentry.Organization")
+    user = FlexibleForeignKey(settings.AUTH_USER_MODEL)
+    date_added = models.DateTimeField(default=timezone.now)
+    date_finished = models.DateTimeField(null=True)
+    date_expired = models.DateTimeField(null=True)
+    storage_url = models.URLField(null=True)
+    query_type = models.PositiveSmallIntegerField()
```

Comment:
what is the `query_type` again?

Reply:
I guess the idea is that if this is used in the a bunch of different places (e.g. Billing Dashboard, Discover v1, Issue Filtering) then that category would be saved here. I initially had it as a CharField, but switched to a IntField for consistency and its easier to add/change categories without migrations